CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Additions
 * Add OffsetPaginatedListDto interface to the pagination module.
+* Add configuration to allow 0 as a pagination limit value (disabled by default - limit=1 as minimum value).
+
+### Defects Corrected
```

Comment:
Not sure why this is showing as an addition here. If I do a local `git diff` against master it shows that this was already there and is not an addition from this branch. I think github is diffing this incorrectly.

Reply:
I think github UI doesn't work well if we do merge without rebase. As was done in this commit https://github.com/cerner/beadledom/pull/118/commits/cffffa1b60e18286d92dcbf1bcf3c5c72d8976c9